### PR TITLE
refactor: Remove usage of `lazy_static!` from chain

### PR DESCRIPTION
**Pull request by pmnoxx**
This is continuation of https://github.com/near/nearcore/issues/5317
This time we will format metrics in `chain/chain/src/metrics.rs` file.

Part of PR: https://github.com/near/nearcore/pull/5321/files

### CODE REVIEW

**Comment by pmnoxx** [comment]
@mm-near Can you take a look?

**Comment by pmnoxx** [comment]
@mm-near Can you take a look?

**Review by bowenwang1996 — COMMENTED** [review]
@mzhangmzz please review this PR

File: chain/client/src/client.rs [hunk 1]
```diff
@@ -629,7 +629,7 @@ impl Client {
             encoded_chunk.chunk_hash().0,
         );
 
-        near_metrics::inc_counter(&metrics::CHUNK_PRODUCED_TOTAL);
+        (metrics::CHUNK_PRODUCED_TOTAL.inc());
```
Comment (mm-near) [inline chain/client/src/client.rs:632]: nit: do we need the surrounding brackets?

File: chain/client/src/client.rs [hunk 2]
```diff
@@ -518,7 +518,7 @@ impl Client {
             seen: to_timestamp(Clock::utc()),
         })?;
 
-        near_metrics::inc_counter(&metrics::BLOCK_PRODUCED_TOTAL);
+        (metrics::BLOCK_PRODUCED_TOTAL.inc());
```
Comment (mm-near) [inline chain/client/src/client.rs:521]: nit: do we need surrounding brackets?

**Review by mm-near — APPROVED** [review]

**Review by mzhangmzz — APPROVED** [review]

**Review by chefsale — APPROVED** [review]

**Pull request merged**